activerecord/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Rails 5.2.0.beta1 (November 27, 2017) ##
```

Comment:
So close 😄 but the new changelog entry needs to be _above_ this line, at the very top of the file.

Rails 5.2.0.beta1 has already been released, so it doesn't make sense to include this change under that heading.